docs/user/upgrade.rst

```diff
@@ -99,6 +99,12 @@ FileSystem Data Store Metadata Format Change
 The metadata format for the FileSystem data store has been changed to support storing arbitrary key-value pairs.
 Any data written with version 3.2.0 or later will not be readable by earlier GeoMesa versions.
 
+Lambda Data Store Binary Distribution Change
```

Comment:
Hmm... any reason this changed with this PR?

Reply:
i had to make a bunch of changes to the dists for nailgun support (adding the ng client, modifying the scripts, updating the -env classes), so I made an archetype that standardizes everything into one place. i forget why, exactly, but that made including the accumulo runtime in the lambda dist problematic. plus, this makes our artifacts smaller :) if you feel strongly, i can try to include it again

Reply:
No need.